Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.10-slim-buster as base
+RUN apt-get update; apt-get install curl -y
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH="/root/.local/bin:$PATH"
+RUN mkdir todo_app
+COPY pyproject.toml /todo_app/pyproject.toml
```

Comment:
You should copy in `poetry.lock` too - that makes sure you'll get exactly the same versions as you have locally.